packages/shadydom/src/patches/css-selector-splitter.js

```diff
@@ -0,0 +1,140 @@
+/**
```

Comment:
This package doesn't handle escape sequences :

```js
console.log('a\\,b,c', splitSelector('a\\,b,c'))
// a\,b,c { selectors: [ 'a\\', 'b', 'c' ], joiners: [ ',', ',' ] }
```

Reply:
This should work better : https://github.com/mrhenry/core-web/blob/main/packages/core-web-generator/polyfills/%7Eelement-qsa-has.js#L270-L378